profile/store.go

```diff
@@ -469,7 +468,7 @@ func (r *LocalStore) GetProfile(id ID) (*Profile, error) {
 		}
 	}
 
-	return nil, qfs.ErrNotFound
+	return nil, ErrNotFound
```

Comment:
Somewhere above this in the stack (maybe @ `inst.activeProfile`) the `profile.ErrorNotFound` error should instead be changed to an "unauthorized request" error or something similar.

Reply:
We had a chat a while ago, one of the things dispatch chould help centralize the error processing and we could potentially translate "internal error states" into something human readable. For now maybe just a more specific error can help - "request profile not set" or something along the lines.